src/v1/Controllers/Item.php

```diff
@@ -397,7 +406,25 @@ public function postItem(Request $request, Response $response, $args): Response
     $item->owner_user_id = 0;
     $item->owner_group_id = 0;
     $item->state_id = 0;
-    $item->save();
+    // To prevent deadlock on heavy charge because have a select in insert for managing id_bytype (see Item model)
+    $max_retries = 4;
+    $retries = 0;
+    while (true)
+    {
+      try {
+        $item->save();
+      }
+      catch (\Exception $e)
+      {
+        if ($retries <= $max_retries)
+        {
+          $retries++;
+          continue;
+        }
+        throw $e;
+      }
+      break;
+    }
```

Comment:
(nitpicking, to avoid `break` and `continue`)

```suggestion
    $loop = true;
    while ($loop)
    {
      try {
        $item->save();
        $loop = false;
      }
      catch (\Exception $e)
      {
        if ($retries > $max_retries)
        {
          throw $e;
        }
        $retries++;
      }
    }
```